### PR TITLE
Tutorial: Add asciinema to cli args chapter (WIP)

### PR DESCRIPTION
**Pull request by killercup**
This is kinda broken as svg, and asciinema is in general unhappy with vim it seems.

### CODE REVIEW

File: src/tutorial/cli-args.md [hunk 1]
```diff
@@ -136,3 +136,22 @@ it will print out an error or help message and exit the program.
...
 </aside>
+
+## This is what it may look like
+
+![](./tutorial/cli-args.svg)
+
+<aside class="todo">
+
+**TODO:**
+Dafuq is wrong with asciinema and vim?
```
Comment (artem-zinnatullin) [inline src/tutorial/cli-args.md]: nit: "What is wrong with asciinema and vim?"

**Comment by killercup** [comment]
went with a simple version using `cat`

**Comment by spacekookie** [comment]
Everybody knows `ed(1)` is the standard unix text editor

**Comment by killercup** [comment]
I read that as 😸 -proval

**Pull request merged**